src/other/esrgan.jl

```diff
@@ -0,0 +1,128 @@
+using Flux
+using Flux:@functor
+
+function ConvBlock(inc,out,k,s,p,use_act)
+    if use_act 
+        return Chain(
+            Conv((k,k),inc => out,stride = s,pad = p,bias=true),
```

Comment:
```suggestion
            Conv((k, k), inc => out, stride = s, pad = p, bias = true),
```
please check the use of whitespace throughout your code

Reply:
thanks!